docker/docker-compose.yml

```diff
@@ -133,9 +133,13 @@ services:
 
   # Mailpit is a local SMTP server, to replace AWS SES in local development
   mailpit:
-    image: axllent/mailpit
+    image: axllent/mailpit:v1.19
```

Comment:
Unrelated, but since I am changing this file I am pinning also mailpit to a specific version

Reply:
You're right, I was just lazy 👼  Moved here: https://github.com/pagopa/interop-be-monorepo/pull/756